src/common/components/exchange/TxSummaryFixed.vue

```diff
@@ -444,6 +446,7 @@
                 {{ summary.fee }}
                 {{ currencyToTicker }}
               </p>
+              <p class="light-grayish text-end">USD {{ safeFeeUSD }}</p>
```

Comment:
Fiat amount text color is slightly different here.

Reply:
We need to maintain USD to inform the correct coin for our user

Reply:
This is another issue, I mean the color diff here:
<img width="375" alt="Screenshot 2023-09-22 at 17 12 07" src="https://github.com/rsksmart/2wp-app/assets/117093501/86fb3a40-0b7a-4e38-b067-4f0379bc3e67">

Reply:
fixed thanks